packages/file_selector/file_selector_platform_interface/test/x_type_group_test.dart

```diff
@@ -7,12 +7,13 @@ import 'package:flutter_test/flutter_test.dart';
 
 void main() {
   group('XTypeGroup', () {
+    final List<String> extensions = <String>['txt', 'jpg'];
+    final List<String> mimeTypes = <String>['text/plain'];
+    final List<String> macUTIs = <String>['public.plain-text'];
+    final List<String> webWildCards = <String>['image/*'];
+
     test('toJSON() creates correct map', () {
       const String label = 'test group';
-      final List<String> extensions = <String>['txt', 'jpg'];
-      final List<String> mimeTypes = <String>['text/plain'];
-      final List<String> macUTIs = <String>['public.plain-text'];
-      final List<String> webWildCards = <String>['image/*'];
```

Comment:
Please revert this change, and just inline whatever values you need in the new tests. Tests should generally be as localized as possible, to make them easy to understand and change in isolation.